croud/api.py

```diff
@@ -93,7 +93,7 @@ def from_args(args: Namespace) -> "Client":
             CONFIG.endpoint,
             token=CONFIG.token,
             on_token=CONFIG.set_current_auth_token,
-            region=args.region,
+            region=args.region if args.region is not None else CONFIG.region,
```

Comment:
If you don't want to explicitly check for `None`, you could simpler write:

```suggestion
            region=args.region or CONFIG.region,
```